cmake/DyninstCapArchDef.cmake

```diff
@@ -6,59 +6,107 @@
 
 include_guard(GLOBAL)
 
-set(CAP_DEFINES -Dcap_dynamic_heap -Dcap_liveness -Dcap_threads)
-
-if(DYNINST_HOST_ARCH_I386)
-  set(ARCH_DEFINES_TESTSUITE -Darch_x86)
-elseif(DYNINST_HOST_ARCH_X86_64)
-  set(ARCH_DEFINES_TESTSUITE -Darch_x86_64 -Darch_64bit)
-elseif(DYNINST_HOST_ARCH_PPC64LE)
-  set(ARCH_DEFINES_TESTSUITE -Darch_power -Darch_64bit)
-elseif(DYNINST_HOST_ARCH_AARCH64)
-  set(ARCH_DEFINES_TESTSUITE -Darch_aarch64 -Darch_64bit)
-endif()
+## amdgpu special
+if(DYNINST_ARCH_amdgpu)
```

Comment:
Use DYNINST_CODEGEN_ARCH_AMDGPU_GFX908 and put it after other ARCH_DEFINES for CODEGEN_ARCHs